pkg/services/sqlstore/migrations/alert_mig.go

```diff
@@ -168,4 +168,8 @@ func addAlertMigrations(mg *Migrator) {
 	mg.AddMigration("Remove unique index org_id_name", NewDropIndexMigration(alert_notification, &Index{
 		Cols: []string{"org_id", "name"}, Type: UniqueIndex,
 	}))
+
+	mg.AddMigration("Add column no_data_flag in alert", NewAddColumnMigration(alertV1, &Column{
+		Name: "no_data_flag", Type: DB_Bool, Nullable: false, Default: "0",
```

Comment:
Don't think `Nullable: false` can be used for those that have existing data in the alert table. At least not for mysql or postgres and or the default of 0 is not gonna work in postgres or mysql I think, but we need to verify this since I'm not sure.

Reply:
good point! I only added since there is no handling for `null` values. Maybe checking for `null` is preferable over updating all the existing alerts.